lua/kubectl/views/nodes/definition.lua

```diff
@@ -71,7 +71,7 @@ function M.getStatus(row)
   if ready and ready.status == "True" then
     return { symbol = hl.symbols.success, value = nodeConditions.NodeReady }
   end
-  return { symbol = events.ColorStatus("Error"), value = "Unknown" }
+  return { symbol = events.ColorStatus("NotReady"), value = "NotReady" }
```

Comment:
I'm thinking we might keep the unknown as the last state and move the not ready into the if ready == true but in the elseif ready and ready.status == "false", what do you think?

Reply:
something like this?
[9cc3d80](https://github.com/Ramilito/kubectl.nvim/pull/490/commits/9cc3d809528c2ed5b92df63093e3054f1f66b425)

Reply:
Exactly like that! Would have written code example but was on phone 😅